app/Swagger/schemas.php

```diff
@@ -351,3 +351,93 @@ class RSVPUpdateRequestSchema_{
     ]
 )]
 class RSVPAdminAddRequestSchema {}
+
+#[OA\SecurityScheme(
+        type: 'oauth2',
+        securityScheme: 'summit_payment_gateway_oauth2',
+        flows: [
+            new OA\Flow(
+                authorizationUrl: L5_SWAGGER_CONST_AUTH_URL,
+                tokenUrl: L5_SWAGGER_CONST_TOKEN_URL,
+                flow: 'authorizationCode',
+                scopes: [
+                    SummitScopes::ReadPaymentProfiles => 'Read Payment Profiles',
+                    SummitScopes::WritePaymentProfiles => 'Write Payment Profiles',
+                ],
+            ),
+        ],
+    )
+]
+class PaymentGatewayAuthSchema{}
+
+#[OA\Schema(
+    schema: 'PaymentGatewayProfile',
+    type: 'object',
+    properties: [
+        new OA\Property(property: 'id', type: 'integer', example: 1),
+        new OA\Property(property: 'created', type: 'integer', format: 'int64', example: 1633024800),
+        new OA\Property(property: 'last_edited', type: 'integer', format: 'int64', example: 1633024800),
+        new OA\Property(property: 'active', type: 'boolean', example: true),
+        new OA\Property(property: 'provider', type: 'string', enum: ['Stripe', 'LawPay'], example: 'Stripe'),
+        new OA\Property(property: 'application_type', type: 'string', enum: ['Registration', 'BookableRooms'], example: 'Registration'),
+        new OA\Property(property: 'test_mode_enabled', type: 'boolean', example: false, description: 'Only for Stripe provider'),
+        new OA\Property(property: 'live_publishable_key', type: 'string', example: 'pk_live_...', description: 'Only for Stripe provider'),
+        new OA\Property(property: 'test_publishable_key', type: 'string', example: 'pk_test_...', description: 'Only for Stripe provider'),
+    ]
+)]
+class PaymentGatewayProfileSchema {}
+
+#[OA\Schema(
```

Comment:
@matiasperrone-exo why are u defining these here ?
u need to create a file schema per controller and define there this as we already spoke